wear/src/main/java/io/homeassistant/companion/android/onboarding/phoneinstall/PhoneInstallView.kt

```diff
@@ -0,0 +1,100 @@
+package io.homeassistant.companion.android.onboarding.phoneinstall
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.Button
+import androidx.wear.compose.material.ButtonDefaults
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.PositionIndicator
+import androidx.wear.compose.material.Scaffold
+import androidx.wear.compose.material.Text
+import androidx.wear.compose.material.rememberScalingLazyListState
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.views.ThemeLazyColumn
+import io.homeassistant.companion.android.common.R as commonR
+
+@Composable
+fun PhoneInstallView(
+    onInstall: () -> Unit,
+    onRefresh: () -> Unit,
+    onAdvanced: () -> Unit
+) {
+    val scrollState = rememberScalingLazyListState()
+    Scaffold(
+        positionIndicator = {
+            if (scrollState.isScrollInProgress) {
+                PositionIndicator(scalingLazyListState = scrollState)
+            }
+        }
+    ) {
```

Comment:
Don't forget to show the time 😃 

https://developer.android.com/docs/quality-guidelines/wear-app-quality#upcoming

`Display the time of day clearly at the top of the app home screen and any ongoing activity screens. We recommend that you display the time of day at the top of all activities except dialogs and confirmation screens.`

Reply:
Is this a home screen? It's not shown on other onboarding screens either.

Reply:
No but they did recommend to show it on any activity 🙃

Reply:
OK add it to this activity now, update others later?

Reply:
yea I thought we had it on all others but seems its missing from entity details view